### ISSUE TITLE
[14.0] wysiwyg blocked by traeffik rules?

### ISSUE THREAD

**Issue by DarioLodeiros** [Tecnativa/doodba#340]
Hi!
In several tests that I have done with doobda in v14 the website editor gives an error when trying to load:

`Traceback:
Error: Wysiwyg is undefined
willStart/</<@https://commitsun.com/web_editor/static/src/js/wysiwyg/root.js:40:21
_.forEach@https://commitsun.com/web/static/lib/underscore/underscore.js:145:17
willStart/<@https://commitsun.com/web_editor/static/src/js/wysiwyg/root.js:39:19`

I am not sure if it is a mistake of mine in the configuration or a crash of some traffik rule (in dev environment it works perfectly)

Thanks a lot !

**Comment by Yajo** [comment]
Duplicate of https://github.com/Tecnativa/doodba-copier-template/issues/166
